errors/__tests__/apiErrors.ts

```diff
@@ -80,77 +80,60 @@ describe('errors/apiErrors', () => {
       ).toBe(false);
       expect(isMissingScopeError(error2)).toBe(false);
     });
-
-    it('handles AxiosError returned in cause property', () => {
-      const axiosError = newAxiosError({
-        response: {
-          status: 403,
-          data: { category: 'BANNED', subCategory: 'USER_ACCESS_NOT_ALLOWED' },
-        },
-      });
-      const error1 = new Error('', { cause: axiosError });
-      expect(
-        isSpecifiedError(error1, {
-          statusCode: 403,
-          category: 'BANNED',
-          subCategory: 'USER_ACCESS_NOT_ALLOWED',
-        })
-      ).toBe(true);
-    });
   });
   describe('isMissingScopeError()', () => {
     it('returns true for missing scope errors', () => {
-      const error1 = newAxiosError({
+      const error1 = newHubSpotHttpError({
         response: { status: 403, data: { category: 'MISSING_SCOPES' } },
       });
       expect(isMissingScopeError(error1)).toBe(true);
     });
 
     it('returns false for non missing scope errors', () => {
-      const error1 = newAxiosError({
+      const error1 = newHubSpotHttpError({
         response: { status: 400, data: { category: 'MISSING_SCOPES' } },
```

Comment:
Do we need to rewrite this test with another error category? Isn't this the test for non-missing scope errors?

Reply:
I see the same pattern below, am I missing something?